### PR TITLE
Add pagerduty notification docs

### PR DESCRIPTION
**Pull request by mohabusama**
Requires: https://github.com/zalando-zmon/zmon-worker/pull/139

### CODE REVIEW

File: docs/user/notifications/pagerduty.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Pagerduty
+---------
+
+Notify `Pagerduty <https://www.pagerduty.com/>`_ of a new alert status. If the alert ended then notification will ``resolve`` pagerduty incident.
```
Comment (hjacobs) [inline docs/user/notifications/pagerduty.rst]: this sounds a bit weird "if the alert ended then notification will `resolve` pagerduty incident."

Comment (mohabusama) [reply docs/user/notifications/pagerduty.rst]: Updated to more detailed description.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Comment by mohabusama** [comment]
👍

**Pull request merged**